### ISSUE TITLE
Support for ruby 2.7.0

### ISSUE THREAD

**Issue by anquegi** [pry/pry-doc#95]
When using ruby 2.7.0 with OSX and RVM i get:

```
➜ pry
ruby/2.7.0 isn't supported by this pry-doc version
<main>:1: warning: __FILE__ in eval may not return location in binding; use Binding#source_location instead
/Users/toni/.rvm/gems/ruby-2.7.0@learn/gems/pry-0.12.2/lib/pry/commands/whereami.rb:40: warning: in `eval'
<main>:1: warning: __LINE__ in eval may not return location in binding; use Binding#source_location instead
/Users/toni/.rvm/gems/ruby-2.7.0@learn/gems/pry-0.12.2/lib/pry/commands/whereami.rb:41: warning: in `eval'
<main>:1: warning: __FILE__ in eval may not return location in binding; use Binding#source_location instead
/Users/toni/.rvm/gems/ruby-2.7.0@learn/gems/pry-0.12.2/lib/pry/method/weird_method_locator.rb:88: warning: in `eval'
<main>:1: warning: __FILE__ in eval may not return location in binding; use Binding#source_location instead
/Users/toni/.rvm/gems/ruby-2.7.0@learn/gems/pry-0.12.2/lib/pry/method/weird_method_locator.rb:80: warning: in `eval'
[1] pry(main)>
```

**Comment by brentgreeff** [comment]
I have replaced `pry-doc` with `pry` in my gemfile, - when I run `guard`, I get:

> guard
22:16:45 - INFO - Guard::RSpec is running
22:16:45 - INFO - Guard is now watching at '/Users/brentgreeff/x/y'
<main>:1: warning: __FILE__ in eval may not return location in binding; use Binding#source_location instead
/Users/brentgreeff/.rvm/gems/ruby-2.7.0@yy/gems/pry-0.12.2/lib/pry/commands/whereami.rb:40: warning: in `eval'
<main>:1: warning: __LINE__ in eval may not return location in binding; use Binding#source_location instead
/Users/brentgreeff/.rvm/gems/ruby-2.7.0@yy/gems/pry-0.12.2/lib/pry/commands/whereami.rb:41: warning: in `eval'
<main>:1: warning: __FILE__ in eval may not return location in binding; use Binding#source_location instead
/Users/brentgreeff/.rvm/gems/ruby-2.7.0@yy/gems/pry-0.12.2/lib/pry/method/weird_method_locator.rb:88: warning: in `eval'
<main>:1: warning: __FILE__ in eval may not return location in binding; use Binding#source_location instead
/Users/brentgreeff/.rvm/gems/ruby-2.7.0@yy/gems/pry-0.12.2/lib/pry/method/weird_method_locator.rb:80: warning: in `eval'

**Comment by kyrylo** [comment]
I just released v1.1.0 with Ruby 2.7 support.
